mod-data-import-converter-storage-server/src/main/java/org/folio/services/association/CommonProfileAssociationService.java

```diff
@@ -105,6 +118,17 @@
     return result;
   }
 
+  /**
+   * Returns ProfileAssociationDao instance according to specified master and detail types
+   *
+   * @param masterType a master type in association
+   * @param detailType a detail type in association
+   * @return ProfileAssociationDao implementation
+   */
+  private ProfileAssociationDao getProfileAssociationDao(ContentType masterType, ContentType detailType) {
+    return (ProfileAssociationDao) applicationContext.getBean(format("%s_TO_%s", masterType.value(), detailType.value()));
```

Comment:
Please move this "%s_TO_%s" into string constant

Reply:
What happens if no bean found by this composite id ? Surround this call to applicationContext with try-catch block, and catch BeanNotFoundException